### PR TITLE
Remove zero init and constant biasing for `to_gate`

### PR DESCRIPTION
**Pull request by amorehead**
Attention pair biasing uses a simple linear (no bias) and a sigmoid function to gate the attention output logits, as shown below 
image

### CODE REVIEW

File: alphafold3_pytorch/attention.py [hunk 1]
```diff
@@ -224,11 +223,7 @@ def __init__(
         self.to_gates = None
 
         if gate_output:
-            gate_linear = nn.Linear(dim, dim_inner)
-            nn.init.zeros_(gate_linear.weight)
-            nn.init.constant_(gate_linear.bias, init_gate_bias)
-
-            self.to_gates = gate_linear
+            self.to_gates = nn.Sequential(nn.Linear(dim, dim_inner, bias = False), nn.Sigmoid())
```
Comment (lucidrains) [inline alphafold3_pytorch/attention.py:226]: want to use the LinearNoBias here?

Comment (amorehead) [reply alphafold3_pytorch/attention.py:226]: Sure

Comment (amorehead) [reply alphafold3_pytorch/attention.py]: Had to duplicate to avoid circular imports

**Review by lucidrains — COMMENTED** [review]
oh my bad, thought this was a constant in this file!

File: alphafold3_pytorch/attention.py (hunk 1 above)
Comment (lucidrains) [reply alphafold3_pytorch/attention.py]: yeah that's fine, dryness is overrated
